xcp_d/cli/run.py

```diff
@@ -665,7 +665,8 @@ def build_workflow(opts, retval):
         retval["return_code"] = 1
 
     # Bandpass filter parameters
-    if opts.bandpass_filter and (opts.lower_bpf >= opts.upper_bpf):
+    if opts.bandpass_filter and (opts.lower_bpf >= opts.upper_bpf) and \
+            (opts.lower_bpf > 0 and opts.upper_bpf > 0):
```

Comment:
It could also be that black flags a style issue, but isn't able to solve it. I think black would generally recommend something like the following though:

```suggestion
    if (
        opts.bandpass_filter and
        (opts.lower_bpf >= opts.upper_bpf) and
        (opts.lower_bpf > 0 and opts.upper_bpf > 0)
    ):
```